src/OpenGraph.php

```diff
@@ -16,8 +16,11 @@ public function fetch($url, $allMeta = null, $lang = null)
         $doc = new DOMDocument();
 
         $libxml_previous_state = libxml_use_internal_errors(true);
-        $doc->loadHTML('<?xml encoding="utf-8" ?>'.$html);
+        $doc->loadHTML('<?xml encoding="utf-8" ?>'.$html, $options);
         //catch possible errors due to empty or malformed HTML
+        if ($options > 0 && ($options & (LIBXML_NOWARNING | LIBXML_NOERROR)) == 0) {
+            Log::warning(libxml_get_errors());
```

Comment:
@alinmiron Thanks for the contribution. For Log::warning I think we need to make change in the composer.json also.

Reply:
I just pushed a commit now